modules/oadp-ceph-cephfs.adoc

```diff
@@ -0,0 +1,167 @@
+* The default `StorageClass` and `VolumeSnapshotClass` CRs are defined for cephFS and OADP 1.2 Data Mover
+
+Procedure
+
+. Verify that `deletionPolicy` of the `VolumeSnapshotClass` CR is set to `Retain`:
++
+[source,terminal]
+----
+$ oc get volumesnapshotclass -A  -o jsonpath='{range .items[*]}{"Name: "}{.metadata.name}{"  "}{"Retention Policy: "}{.deletionPolicy}{"\n"}{end}'
+----
+
+. Verify that the `VolumeSnapshotClass` CR labels are set to `true`:
++
+[source,terminal]
+----
+$ oc get volumesnapshotclass -A  -o jsonpath='{range .items[*]}{"Name: "}{.metadata.name}{"  "}{"labels: "}{.metadata.labels}{"\n"}{end}'
+----
+
+. Verify that the `StorageClass` annotations are as expected:
++
+[source,terminal]
+----
+$ oc get storageClass -A  -o jsonpath='{range .items[*]}{"Name: "}{.metadata.name}{"  "}{"annotations: "}{.metadata.annotations}{"\n"}{end}'
+----
+
+. Create a Data Protection Application (DPA) CR similar to one below. Use the Restic `Secret` you created when you prepared your environment for working with OADP 1.2 Data Mover and Ceph as the value for `spec.features.dataMover.credentialName`. If you do not, then the CR will use the default value `dm-credential` for this parameter. Note that there is no default value for `enable` label. Each one can have a value of `true` or `false`.
++
+Example DPA CR
++
+[source,yaml]
+----
+apiVersion: oadp.openshift.io/v1alpha1
+kind: DataProtectionApplication
+metadata:
+  name: velero-sample
+  namespace: openshift-adp
+spec:
+  backupLocations:
+    - velero:
+        config:
+          profile: default
+          region: us-east-1
+        credential:
+          key: cloud
+          name: cloud-credentials
+        default: true
+        objectStorage:
+          bucket: <my-bucket>
+          prefix: velero
+       provider: aws
+    configuration:
+      restic:
+        enable: false
+      velero:
+        defaultPlugins:
+          - openshift
+          - aws
+          - csi
+          - vsm
+    features:
+      dataMover:
+        credentialName: <restic-secret-name>
+        enable: true
+        volumeOptionsForStorageClasses:
+          ocs-storagecluster-cephfs:
+            sourceVolumeOptions:
+              accessMode: ReadOnlyMany
+              cacheAccessMode: ReadWriteMany
+              cacheStorageClassName: ocs-storagecluster-cephfs
+              storageClassName: ocs-storagecluster-cephfs-shallow
+----
+
+. To back up data:
+
+.. Create a `Backup` CR:
++
+Example `Backup` CR
++
+[source,yaml]
+----
+apiVersion: velero.io/v1
+kind: Backup
+metadata:
+  name: <backup-name>
+  namespace: <protected-ns>
+spec:
+  includedNamespaces:
+- <app-ns>
+storageLocation: velero-sample-1
+----
+
+.. Monitor the Data Mover backup and its artifacts by running the script named link:https://github.com/openshift/oadp-operator/blob/master/docs/examples/datamover_resources.sh[Volumemover live debug]
```

Comment:
In general, we are not to link out to GitHub scripts. Has this been validated by QE?

See [this thread](https://redhat-internal.slack.com/archives/C85JYPHL3/p1686057008204659) for more discussion on this topic, including concerns about scripts on the `main`/`master` branch.
```suggestion
.. Monitor the Data Mover backup and its artifacts by running the script named link:https://github.com/openshift/oadp-operator/blob/master/docs/examples/datamover_resources.sh[Volumemover live debug].
```

Reply:
@jeana-redhat Good thing you asked -- QE vetted the PR but when i asked about the script specifically, the lead engineer replied that the script is not for downstream. Will remove it when I fix this PR.